baselines/disable-packing-test/protos/google/showcase/v1beta1/identity.proto.baseline

```diff
@@ -102,24 +102,23 @@ message User {
     (google.api.field_behavior) = OUTPUT_ONLY
   ];
 
-  // Typescript generator doe not support proto3 optional fields
   // The age of the use in years.
-  // optional int32 age = 6;
+  optional int32 age = 6;
```

Comment:
This is interesting :)  I forgot to update this baseline proto file in #539 but the tests did not fail. Do we ignore proto files when we compare baselines?

Reply:
I think you updated this file in that PR, at least from here:
https://github.com/googleapis/gapic-generator-typescript/pull/539/files#diff-755cd9dd88b3024c1452c8792a91db45L107

Reply:
No, I updated `baselines/showcase` but not `baselines/disable-packing-test`. It should have failed but it did not :)  This is unrelated to this PR, but seems like the baseline test does not always detect proto files changes for whatever reason.

Reply:
ummmm, I see, that's also updated from another PR, when I run `npm run baseline`, it updates the library.proto.baseline.